libs/tree-sitter-titan/grammar.js

```diff
@@ -1,7 +1,42 @@
 module.exports = grammar({
-  name: 'titan',
-
+  name: "titan",
   rules: {
```

Comment:
this way we can support multiple statements in a single file

```suggestion
  rules: {
    source_file: $ => repeat($._statement),

    _statement: ($) => choice($.variable_definition),
```